src/quacc/recipes/emt/md.py

```diff
@@ -0,0 +1,69 @@
+"""
+Molecular Dynamics recipes for EMT.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from ase.calculators.emt import EMT
+
+from quacc import job
+from quacc.runners.ase import Runner
+from quacc.schemas.ase import summarize_md_run
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from ase.atoms import Atoms
+
+    from quacc.schemas._aliases.ase import DynSchema
+    from quacc.utils.files import Filenames, SourceDirectory
+
+
+@job
+def md_job(
+    atoms: Atoms,
+    md_params: dict[str, Any] | None = None,
+    copy_files: SourceDirectory | dict[SourceDirectory, Filenames] | None = None,
+    **calc_kwargs,
+) -> DynSchema:
+    """
+    Carry out a Molecular Dynamics calculation.
+
+    !!! Important "Units"
+
+        Quacc does not follow ASE standards for Molecular Dynamics units and instead
+        uses the following units:
+
+        - Time: femtoseconds (fs)
+        - Pressure: GPa
+        - Temperature: Kelvin (K)
+        - Compressibility: 1/GPa
```

Comment:
I do wonder if this is going to introduce additional confusion simply because if people don't read the docstring (common), it'll just lead to a confusing mess. At the same time, the ASE units themselves are disgusting. I'm a bit torn.